controllers/finalizers.go

```diff
@@ -320,3 +337,26 @@ func (r *MultiClusterHubReconciler) cleanupFoundation(reqLogger logr.Logger, m *
 
 	return nil
 }
+
+func (r *MultiClusterHubReconciler) orphanOwnedMultiClusterEngine(m *operatorsv1.MultiClusterHub) error {
+	ctx := context.Background()
+
+	managedByMCE, err := r.ManagedByMCEExists()
+	if err != nil {
+		return err
+	}
+	if managedByMCE == nil {
+		// MCE does not exist
+		return nil
+	}
+	r.Log.Info("Preexisting MCE exists, orphaning resource")
+	managedByMCE.Finalizers = utils.RemoveString(managedByMCE.Finalizers, hubFinalizer)
```

Comment:
Controller runtime has some utility functions you can take advantage of for common use cases such as this, like https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/controller/controllerutil#RemoveFinalizer, seen used in https://book.kubebuilder.io/reference/using-finalizers.html